erpnext/stock/get_item_details.py

```
@@ -1209,12 +1209,40 @@ def get_price_list_rate_for(ctx: ItemDetailsCtx, item_code):
+			item_uoms = frappe.get_all(
+				"UOM Conversion Detail",
+				filters={"parent": item_code, "parenttype": "Item"},
+				fields=["uom", "conversion_factor"],
+			)
+			for uom_row in item_uoms:
+				if uom_row.uom not in [ctx.get("uom"), ctx.get("stock_uom")]:
+					pctx.uom = uom_row.uom
+					alt_price = get_item_price(pctx, item_code, ignore_party=ctx.get("ignore_party"))
+					if alt_price:
+						# Convert the price from the found UOM to the requested UOM
+						requested_uom = ctx.get("uom") or ctx.get("stock_uom")
```

Ruff (0.14.14)

[error] Local variable `requested_uom` is assigned to but never used

Remove assignment to unused variable `requested_uom`

(F841)